app/src/layers/containers/CustomLayer.js

```diff
@@ -1,22 +1,40 @@
 import { connect } from 'react-redux';
 import CustomLayer from 'layers/components/CustomLayer';
 import { setLayerManagementModalVisibility } from 'app/appActions';
-import { uploadCustomLayer } from 'layers/customLayerActions';
+import { resetCustomLayerForm, uploadCustomLayer, confirmCustomLayer } from 'layers/customLayerActions';
 import { login } from 'user/userActions';
 
+const getSubLayers = ({ capabilities }) => {
+  const layers = capabilities && capabilities.Capability && capabilities.Capability.Layer && capabilities.Capability.Layer.Layer;
```

Comment:
IMHO the container should not have to deal with WMS-specific stuff. This belongs to the custom layer reducer or actions.

Reply:
You were right, this is now done in actions.